src/quantum/config.js

```diff
@@ -62,4 +65,89 @@ const CONFIG = {
 
 };
 
-export default CONFIG;
+
+const PLATFORMS = [
+  {
+    browser: 'Firefox',
+    bits: '32',
+    os: 'win',
+    label: 'Firefox',
+    frameworkId: 10,
```

Comment:
I will keep reading the code but I believe we could group the perfherder attributes in an object:
```
seriesConfig: {
    frameworkId: 10,
    platform: 'windows10-64',
    option: 'pgo',
    project: 'mozilla-central',
}
```

Reply:
You may be right, but I can not see the need yet.  If we have a `seriesConfig`, then it must also include the `suite` found in the `config.js/SUITES` table.  This will complicate the SUITES table, and it will complicate the `join` of SUITES and PLATFORMS that makes up TP6_PAGES.

I suggest your desire to group the perfherder attributes (which may be legitimate) is a side effect of the shape `<PerfherderGraphContainer>` expects from its parameters.  If `<PerfherderGraphContainer>` parameters were flat, then would you find the flatness of TP6_PAGES acceptable?

The inner-object structure is more difficult for the list comprehensions to manipulate in general.  More complicated structures require more complicated code to manipulate it.  Furthermore, this data structure is particular to `<PerfherderGraphContainer>`, and not general enough, or old enough, to be treated like black box yet.  I suggest it is premature to complicate `PLATFORMS` data structure.

Reply:
I have no problem to keeping it flat.